judo-tatami-jsl-jsl2psm/src/main/epsilon/transformations/psm/modules/structure/transferAttribute.etl

```diff
@@ -36,6 +37,7 @@ rule CreateTransferAttributeForDefaultTransferObjectType
 // the caller code have to add it to the corresponding transfer object
 // This method can be used to make clone attributes to inherited transfer objects
 @lazy
+@greedy
```

Comment:
both lazy and greedy?

Reply:
Thats not opposite things.
- lazy means it is executed when some code calls equivalent for this rule
- greedy means when there is same source / target type mapping for same instance it have to be executed.